controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -2144,6 +2201,104 @@ int cli_bml::get_dcs_scan_results(const std::string &radio_mac, uint32_t max_res
     return 0;
 }
 
+/**
+ * get all client list.
+ * 
+ * @return 0 on success.
+ */
+int cli_bml::client_get_client_list()
+{
+    char client_list[256];
+    unsigned int client_list_size;
+
+    int ret = bml_client_get_client_list(ctx, client_list, &client_list_size);
+    auto client_list_vec =
+        beerocks::string_utils::str_split(std::string(client_list, client_list_size), ',');
+
+    std::cout << "client list:" << std::endl;
+    for (const auto &client : client_list_vec) {
+        std::cout << "- " << client << std::endl;
+    }
+
+    printBmlReturnVals("bml_client_get_client_list", ret);
+
+    return 0;
+}
+
+/**
+ * Set specific client according to MAC.
+ * 
+ * @param [in] sta_mac MAC address of requested client.
+ * @param [in] selected_bands comma-seperated selected bands.
+ * @param [in] stay_on_initial_radio Whather to stay on initial radio or not.
+ * @param [in] stay_on_selected_device Whather to stay on selected device or not.
+ * @return 0 on success.
+ */
+int cli_bml::client_set_client(const std::string &sta_mac, int8_t selected_bands,
```

Comment:
see my comment above regarding optional parameters and how you enforce here a true/false even though the `BML_CLIENT_CONIG` supports optional params

Reply:
Changed to use optional parameters